index.html

```diff
@@ -17,46 +23,60 @@
                 // hint: these are strings
                 var numberOne = document.forms.inputs.numberOne.value;
                 var numberTwo = document.forms.inputs.numberTwo.value;
-                document.forms.inputs.result.value = numberOne + numberTwo;
+                document.forms.inputs.result.value = parseInt(numberOne) + parseInt(numberTwo);
             },
             subtractNumbers: function () {
-                
+                var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) - parseInt(numberTwo);
+            },
+            multiplyNumbers: function () {
+                var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) * parseInt(numberTwo);
+            },
+            divideNumbers: function () {
+                var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) / parseInt(numberTwo);
             }
         }
 
     </script>
 
 </head>
 
-<body>
-    <div>Header</div>
+<body style="background-color: rgb(148, 109,221);">
+    <h1 style="color: rgb(114, 12, 12); text-align: center;">Enter your numbers into the fields below and hit the corresponding button for addition or subtraction!</h1>
```

Comment:
Consider moving the line `style` declaration to a new `h1` selector in the `style` tag.  Putting the selector and style configuration into the style tag allows the "cascading" to occur.  Rather than just this single `h1` having this style, all `h1` tags can be styled by using the style tag configuration.